HW03/Nastja_Kireeva/Nastja_Kireeva_hw_03_t_03.rb

```diff
@@ -0,0 +1,21 @@
+require 'time'
+TIMESTAMP = /^\d{4}-\d{2}-\d{2}\s\d{2}:\d{2}:\d{2}.\d/.freeze
+
+def task3(logs)
+  lines = logs.split(/\n/)
+  events = []
+  lines.map { |line| events << line[TIMESTAMP] if line.include?('Calling') }
+  if events.size <= 1
+    p '0'
+  else
+    calculate_diff(events)
+  end
```

Comment:
One line is preferable if possible
```suggestion
  events.empty? ? 0 : calculate_diff(events)
```